src/alks.js

```diff
@@ -768,6 +768,19 @@ class alks {
       delete opts.password
     }
 
+    if( opts.accessKey ) {
+      headers['ALKS-STS-Access-Key'] = opts.accessKey;
```

Comment:
This line ends with a `;` which is breaking as `headers` is an object. Don't need the semicolons in any of the setters.